setup.py

```diff
@@ -66,189 +76,192 @@ class BinaryDistribution(Distribution):
-        author_email='ama6fy@virginia.edu',
-        description='A Python Wrapper for the NAIF CSPICE Toolkit',
-        long_description=readmetext,
-        keywords=['spiceypy', 'spice', 'naif', 'jpl', 'space', 'geometry'],
-        url='https://github.com/AndrewAnnex/SpiceyPy',
-        classifiers=[
-            "Development Status :: 4 - Beta",
-            "Natural Language :: English",
-            "Topic :: Scientific/Engineering",
-            "Topic :: Scientific/Engineering :: Astronomy",
-            "License :: OSI Approved :: MIT License",
-            "Programming Language :: Python :: 2.7",
-            "Programming Language :: Python :: 3.3",
-            "Programming Language :: Python :: 3.4",
-            "Programming Language :: Python :: 3.5",
-            "Programming Language :: Python :: 3.6",
-            "Operating System :: MacOS :: MacOS X",
-            "Operating System :: POSIX :: Linux",
-            "Operating System :: POSIX :: FreeBSD",
-            "Operating System :: Microsoft :: Windows"
-        ],
-        packages=find_packages(exclude=["*.tests"]),
-        include_package_data=True,
-        zip_safe=False,
-        distclass=BinaryDistribution,
-        package_data={'': ['*.so', "*.dll"]},
-        setup_requires=DEPENDENCIES,
-        install_requires=DEPENDENCIES,
-        requires=REQUIRES,
-        tests_require=TEST_DEPENDENCIES,
-        cmdclass={'test': PyTest},
-        test_suite='spiceypy.tests.test_wrapper.py',
-        extras_require={'testing': ['pytest']}
-    )
-finally:
-    pass
-
+            self.move_to_root_directory()
+
+
+readme = open('README.rst', 'r')
+readmetext = readme.read()
+readme.close()
+
+setup(
+    name='spiceypy',
+    version='2.0.1.dev2',
+    license='MIT',
+    author='Andrew Annex',
+    author_email='ama6fy@virginia.edu',
+    description='A Python Wrapper for the NAIF CSPICE Toolkit',
+    long_description=readmetext,
+    keywords=['spiceypy', 'spice', 'naif', 'jpl', 'space', 'geometry'],
+    url='https://github.com/AndrewAnnex/SpiceyPy',
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "Natural Language :: English",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Scientific/Engineering :: Astronomy",
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3.3",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Operating System :: MacOS :: MacOS X",
+        "Operating System :: POSIX :: Linux",
+        "Operating System :: POSIX :: FreeBSD",
+        "Operating System :: Microsoft :: Windows"
+    ],
+    packages=find_packages(exclude=["*.tests"]),
+    include_package_data=True,
+    zip_safe=False,
+    distclass=BinaryDistribution,
+    package_data={'': ['*.so', "*.dll"]},
+    setup_requires=[],
```

Comment:
I think this should be set back to `DEPENDENCIES`, unless there is a reason not to

Reply:
There are two reasons for not setting it back to DEPENDENCIES. The second one is that setup does not require any of those packages to be installed. Now it deals with a class that imports the required modules only if needed and when install is called. The second one is that they do not play nicely together, particularly for numpy and cryptography (or any other package that may require compiling).